graph/telemetry/istio/appender/ambient.go

```diff
@@ -50,7 +50,7 @@ func (a AmbientAppender) handleWaypoints(trafficMap graph.TrafficMap, globalInfo
 	// Fetch the waypoint workloads
 	waypoints := globalInfo.Business.Workload.GetWaypoints(context.Background())
 
-	deletedWaypoints := make(map[string]bool)
+	waypointsNodes := make(map[string]bool)
```

Comment:
```suggestion
	waypointNodes := make(map[string]bool)
```

Just removing the double-plural.